main.go

```diff
@@ -47,6 +47,10 @@ func main() {
 		app.WithAWSConfig(cfg),
 	}
 
+	if os.Getenv("ELASTIC_APM_LAMBDA_DISABLE_LOGS_API") == "true" {
```

Comment:
Should we use `strconv.ParseBool(...)` to be consistent with ELASTIC_APM_LAMBDA_CAPTURE_LOGS?